### PR TITLE
Abstract constants that change between dev and prod into configuration file.

### PR DESCRIPTION
**Pull request by akbiggs**
Before running cabal build, and after any changes are made to the configuration file, the chosen configuration file needs to be copied from `<ENVIRONMENT>_Config.hs` to `Config.hs`. This file will then be used for the settings in the environment. This allows us to override the configuration variables with different values in production.

### CODE REVIEW

**Comment by david-yz-liu** [comment]
Thanks, @akbiggs. Couple things:

1. Please remove `fbdbStr` from the config file and just put it into `CourseographyFacebook`.
2. Please add some Haddock documentation for the Config module and each value
3. Please move the other path-related strings into this file.

**Comment by akbiggs** [comment]
@david-yz-liu Done

**Comment by david-yz-liu** [comment]
Thanks, @akbiggs!

File: hs/Development_Config.hs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+{-
```
Comment (david-yz-liu) [inline hs/Development_Config.hs]: @akbiggs Didn't notice earlier, but if you want Haddock to know about this, you need to use `{-|`

**Pull request merged**